config/ant-phase-verify.xml

```diff
@@ -147,6 +147,8 @@
          <exclude name="**/InputMainFrameModelIncorrectClass.java"/>
          <exclude name="**/InputBeforeExecutionExclusionFileFilterIncorrectClass.java"/>
          <exclude name="**/InputJavaParser.java"/>
+         <exclude name="**/SuppressionXpathRegressionOuterTypeFilename1.java"/>
+         <exclude name="**/SuppressionXpathRegressionOuterTypeFilename2.java"/>
```

Comment:
Please explain the changes to this file. We weren't expecting them.

Reply:
ok, Because if I don't change the file, a failure will happen when I run mvn clean verify. So I change it according the error information, error details as follows : 

[INFO] --- maven-antrun-plugin:1.8:run (ant-phase-verify) @ checkstyle ---
[INFO] Executing tasks

main:

execute:
     [echo] Checkstyle started (checkstyle_checks.xml): 14/03/2020 08:01:21 PM
[checkstyle] Running Checkstyle  on 1115 files
     [echo] Checkstyle finished (checkstyle_checks.xml) : 14/03/2020 08:02:04 PM
     [echo] Checkstyle started (checkstyle_non_main_files_checks.xml): 14/03/2020 08:01:21 PM
[checkstyle] Running Checkstyle  on 1954 files
     [echo] Checkstyle finished (checkstyle_non_main_files_checks.xml): 14/03/2020 08:02:04 PM
     [echo] Checkstyle started (checkstyle_resources_checks.xml): 14/03/2020 08:01:21 PM
[checkstyle] Running Checkstyle  on 1700 files
[checkstyle] [ERROR] /Users/hgh/Desktop/checkstyle/src/it/resources/org/checkstyle/suppressionxpathfilter/outertypefilename/SuppressionXpathRegressionOuterTypeFilename1.java:3:1: The name of the outer type and the file do not match. [OuterTypeFilename]
[checkstyle] [ERROR] /Users/hgh/Desktop/checkstyle/src/it/resources/org/checkstyle/suppressionxpathfilter/outertypefilename/SuppressionXpathRegressionOuterTypeFilename2.java:3:1: The name of the outer type and the file do not match. [OuterTypeFilename]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 02:54 min
[INFO] Finished at: 2020-03-14T20:02:09+08:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-antrun-plugin:1.8:run (ant-phase-verify) on project checkstyle: An Ant BuildException has occured: The following error occurred while executing this line:
[ERROR] /Users/hgh/Desktop/checkstyle/config/ant-phase-verify.xml:138: Got 2 errors and 0 warnings.
[ERROR] around Ant part ...<ant antfile="config/ant-phase-verify.xml"/>... @ 8:47 in /Users/hgh/Desktop/checkstyle/target/antrun/build-main.xml
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException

If it is not a good way to solve this problem, please give me some advices, thanks.

Reply:
Ok. You cannot exclude the entire file. You must suppress the violations.
https://checkstyle.org/config_filters.html

See https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle_non_main_files_suppressions.xml

Reply:
ok, done and accurately it should be configured in config/checkstyle_resources_suppressions.xml